lib/attrsets.nix

```
@@ -419,18 +419,44 @@ rec {
     set:
     listToAttrs (concatMap (name: let v = set.${name}; in if pred name v then [(nameValuePair name v)] else []) (attrNames set));
 
+  /*
+    An alias for [`filterAttrsRecursiveTopDown`](#function-library-lib.attrsets.filterAttrsRecursiveTopDown).
+  */
+  filterAttrsRecursive = filterAttrsRecursiveTopDown;
 
-  /* Filter an attribute set recursively by removing all attributes for
-     which the given predicate return false.
+  /*
+    Filter an attribute set recursively by removing all attributes for which the given predicate returns `false`.
 
```

This is the defining feature of this function, so I'd put it early.

```suggestion

    Unlike `filterAttrsRecursiveBottomUp`, the value passed to the predicate function is unfiltered.

```